app/views/pages/index.html.erb

```diff
@@ -1,21 +1,52 @@
-<div class="container text-center">
-  <h1>Pages#index</h1>
-  <p>Find me in app/views/pages/index.html.erb</p>
+<div class="container">
   <% if current_user %>
-    <h2>Logged-in User:</h2>
+    <div class="row">
+      <div class="col col-lg-4">
+        <%= link_to 'Create Bet', new_bet_path %>
+      </div>
+      <div class="col col-lg-4"></div>
+      <div class="col col-lg-3"></div>
+      <div class="col col-lg-1">
+        <div class="dropdown">
+          <img type="button" data-toggle="dropdown" src="http://fuuse.net/wp-content/uploads/2016/02/avatar-placeholder.png" style="width: 100%;">
+          <ul class="dropdown-menu">
+            <li><%= link_to 'Log Out', session_path, method: :delete %></li>
```

Comment:
Since this is content that is likely shared across many pages, It should be extracted into a partial and rendered in the application layout